Ginger/Ginger/ALM/Qtest/TreeViewItems/QtestCycleTreeItem.cs

```diff
@@ -0,0 +1,104 @@
+#region License
+/*
+Copyright © 2014-2019 European Support Limited
+
+Licensed under the Apache License, Version 2.0 (the "License")
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at 
+
+http://www.apache.org/licenses/LICENSE-2.0 
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS, 
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+See the License for the specific language governing permissions and 
+limitations under the License. 
+*/
+#endregion
+
+using Ginger.SolutionWindows.TreeViewItems;
+using GingerWPF.UserControlsLib.UCTreeView;
+using GingerCore;
+using System;
+using System.Collections.Generic;
+using System.Windows.Controls;
+using Amdocs.Ginger.Common;
+using amdocs.ginger.GingerCoreNET;
+using GingerCore.ALM.Qtest;
+
+namespace Ginger.ALM.Qtest.TreeViewItems
+{
+    public class QtestCycleTreeItem : TreeViewItemBase, ITreeViewItem
+    {
+        public BusinessFlow QCExplorer { get; set; }
+        public List<ITreeViewItem> currentChildrens = null;
+
+        public string ID { get; set; }
+        public string Name { get; set; }
+        public string Path { get; set; }
+
+        Object ITreeViewItem.NodeObject()
+        {
+            return null;
+        }
+
+        public QtestCycleTreeItem(List<QTestApiModel.TestCycleResource> childrenCycles = null, List<QTestApiModel.TestSuiteWithCustomFieldResource> childrenSuites = null)
+        {
+            currentChildrens = new List<ITreeViewItem>();
+
+            if (childrenCycles != null)
+            {
+                foreach (QTestApiModel.TestCycleResource cycle in childrenCycles)
+                {
+                    QtestCycleTreeItem pfn = new QtestCycleTreeItem(cycle.TestCycles, cycle.TestSuites);
+                    pfn.Name = cycle.Name.ToString();
+                    pfn.ID = cycle.Id.ToString();
+                    pfn.Path = cycle.Path.ToString();
+                    currentChildrens.Add(pfn);
+                }
+            }
+
+            if (childrenSuites != null)
+            {
+                foreach (QTestApiModel.TestSuiteWithCustomFieldResource suite in childrenSuites)
+                {
+                    QtestSuiteTreeItem pfn = new QtestSuiteTreeItem();
+                    pfn.Name = suite.Name.ToString();
+                    pfn.ID = suite.Id.ToString();
+                    currentChildrens.Add(pfn);
+                }
+            }
+        }
+
+        StackPanel ITreeViewItem.Header()
+        {
+            return TreeViewUtils.CreateItemHeader(Name, "@WorkFlow_16x16.png");
+            // return TreeViewUtils.CreateItemHeader(Name, "@TestSet_16x16.png");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5395796)